api-module-library/ironclad/test/api.test.js

```diff
@@ -223,19 +173,19 @@ describe('Ironclad API class', () => {
 
         it('should list all record schemas', async () => {
             const response = await api.listAllRecordSchemas();
-            expect(response).to.have.property('list');
+            expect(response).to.have.property('properties');
+            expect(response).to.have.property('recordTypes');
         });
 
         it('should retrieve a record', async () => {
-            const response = await api.retrieveWorkflow(recordID);
+            const response = await api.retrieveRecord(recordID);
             expect(response).to.have.property('id');
             expect(response).to.have.property('type');
             expect(response).to.have.property('name');
             expect(response).to.have.property('lastUpdated');
             expect(response).to.have.property('properties');
             expect(response).to.have.property('attachments');
             expect(response).to.have.property('links');
-            expect(response).to.have.property('source');
         });
```

Comment:
I suggest adding a test that deletes the created record. Since we're making live tests to the API, we probably want to clean up anything that we've added/modified while running our tests.